### ISSUE TITLE
[syncfusion_flutter_pdfviewer] pdfviewer not scrolling in proportion with zoom in horizontal direction

### ISSUE THREAD

**Issue by sagarkardani** [syncfusion/flutter-widgets#333]
### Issue
- Scrolling behaviour is working fine in vertical direction.
- In horizontal direction it is not scrolling in proportion with zoom.

### Current behaviour

https://user-images.githubusercontent.com/60609139/128015430-cf435246-f8ca-4ce5-8164-bd411d1897a7.mp4

### Expected behaviour

https://user-images.githubusercontent.com/60609139/128015571-552ccb5e-5872-43fd-a4d3-b683a15799af.mp4

**Comment by deepika134** [comment]
Hi sagarkardani,
 
We can reproduce the reported issue from our side. We are analyzing on this and will update the further details on 6 August 2021.
 
Regards,
Deepika R

**Comment by dillibabun** [comment]
Hi @sagarkardani 

Syncfusion Flutter PdfViewer uses InteractiveViewer widget internally to perform zooming and panning in a vertical, horizontal, and diagonal direction. On further analysis, we found that the reported behavior occurs due to InteractiveViewer. As the scrolling momentum is fully controlled by InteractiveViewer, we are unable to override this behavior.

Regards,
Dilli babu.

**Comment by sagarkardani** [comment]
Hi @dillibabun

Thanks for quick reply.

Overall SfPdfViewer is a very great plugin and works very well.

What I noticed is that panning, zooming and scrolling behaviour is not going as 'fluidic' and 'smooth' as compared to other pdfviewers. Probably because in SfPdfViewer that behaviour is being managed by InteractiveViewer widget.

I did some analysis of source codes. Can `onInteractionStart`, `onInteractionUpdate`, `onInteractionEnd` and `transformationController` be tweaked somehow and be made more custom to make all behaviours more smooth and more user friendly ?

I know that implementing this things manually are much complicated requires much effort. But can this be planned as enhancement features giving less priority ?

Regards,
Sagar
